src/dotnet/APIView/APIView/Model/CodeFile.cs

```diff
@@ -186,6 +186,9 @@ public void ConvertToTreeTokenModel()
             List<ReviewToken> currentLineTokens = new List<ReviewToken>();
             foreach(var oldToken in Tokens)
             {
+                //Don't include documentation in converted code file due to incorrect documentation formatting used in previous model.
```

Comment:
how did you find out this discrepancy? are there any automated tests we can run to keep this tool up to date with the design (as it changes)?

Reply:
Also cant you convert the old documentation to new. Seem like that would be a straightforward mapping?

Reply:
> how did you find out this discrepancy? are there any automated tests we can run to keep this tool up to date with the design (as it changes)?

We won't need this conversion once a language is migrated and usage of this tool is to convert an existing review file to new model, specifically C++ and TypeSpec reviews that doesn't have source files.